src/components/generic/movies/style.tsx

```diff
@@ -11,7 +11,7 @@ const Movies = styled.div`
   justify-content: space-evenly;
 `
 
-const img = styled.div`
+export const img = styled.div<{src:string}>`
```

Comment:
It's best practice to have component names to always start with a capital letter. This applies to all the component declarations below.

Reply:
Done  :white_check_mark: